docs/source/histoqc.md

```diff
@@ -0,0 +1,72 @@
+# Instructions to setup and run HistoQC on H&E slides datasets
+
+## Install Openslide
+
+```shell
+sudo apt-get install openslide-tools
+sudo apt-get install python3-openslide
+```
+
+## Clone HistoQC
+
+```shell
+git clone https://github.com/choosehappy/HistoQC
+```
+
+Then edit `HistoQC/environment.devenv.yml` and remove line 17
+
+Create a Conda environment called `histoqc` by running
+
+```shell
+conda env create --file environment.devenv.yml
+conda activate histoqc
+```
+
+Then install the full repo as a Python package in editable mode:
+
+```shell
+pip install -e .
+```
+
+## Install Blobfuse
+
+Get blobfuse to access slides datasets:
```

Comment:
Specify that this is useful for data available on the cloud, if data is predownloaded in the machine, we can skip this step?

Reply:
good point, added some text above.